### ISSUE TITLE
New Token--[Cities Chile]

### ISSUE THREAD

**Issue by mentaer** [hotosm/oam-uploader-admin#18]
Name: Stefan Steiniger
Email: ssteiniger@uc.cl
Location of imagery: Chile
Source of imagery: Ministry of Economy, Project CEDIZ
Short description of collection: aerial imagery for several cities
Have you received approval for making this imagery available (yes/no)?: no, imagery is for download from www.IDE.cl

**Comment by mentaer** [comment]
actually what would be the format. I have the images as TIFF, pretty large though. But the originals come as ecw.

**Comment by smit1678** [comment]
@mentaer Yes, TIFF is fine and what is required right now. 

> Have you received approval for making this imagery available (yes/no)?: no, imagery is for download from www.IDE.cl

Is there a license listed on the site for the data?

**Comment by mentaer** [comment]
No, I have asked them also to do so. But as long as they don't I simply assume its Public Domain as its a Chilean Gov page (could be also CC 4.0 though as used at the open data web page of the government). I would understand if this is problematic for you guys for the moment.
